fr/px-platform.json

```diff
@@ -508,8 +508,8 @@
             },
             "studyEntrance": {
                 "notVerified": {
-                    "title": "Pour participer, vous devez connecter votre compte PayPal vérifié à IntelliZoom.",
-                    "description": "Si vous avez déjà connecté vos comptes, rappelez-vous que pour participer aux tests, vous devez soit passer le test pratique, soit [vérifier votre compte](https://www.paypal.com/uk/smarthelp/article/how-do-i-verify-my-paypal-account-faq444) dans PayPal.",
+                    "title": "",
+                    "description": "",
```

Comment:
That's ok? i guess they do not have transaltion for this still

Reply:
Yes, I guess it is

Reply:
Hmm, maybe I would ask Robi to verify if we can merge it in this state?

Reply:
looks like there a some countries transaltion that are needed to be merged. If this is not going to be solved soon, we can manuall edit this branch to ignore this change and merge the rest

Reply:
Don't think ours is super urgent as the feature is still behind a FT but it would be nice to have these countries merged in this sprint.

Reply:
it's fine now